platform/cosmos/stake.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 const (
-	lockTime      = 1814400
+	lockTime      = 1814400 // in seconds (21 days)
```

Comment:
Having a redundant number in comment is dangerous, can go out of sync.  Instead I suggest:
	lockTime      = 21 * 24 * 3600 // in seconds

Reply:
I will make this change when will refactor staking platforms. This value doesn't used in any calculation.